python/Ganga/new_tests/TestProxy.py

```diff
@@ -69,9 +70,14 @@ def test_dict_attributes(self):
 
         # Proxy class instance
         self.assertFalse(hasattr(self.p, '_proxyClass'))
-        self.assertFalse(hasattr(self.p, '_proxyObject'))
+        self.assertTrue(hasattr(self.p, '_proxyObject'))
         self.assertTrue(hasattr(self.p, '_impl'))
 
+        # Proxied class
+        self.assertTrue(hasattr(self.p_class, '_proxyClass'))
+        self.assertFalse(hasattr(self.p_class, '_proxyObject'))
+        self.assertFalse(hasattr(self.p_class, '_impl'))
+
```

Comment:
Above self.p is an initialized instance self.p_class is a raw class unless we expect for this to be evaluated into an instance?